.github/workflows/aws-lambda-java-runtime-interface-client.yml

```diff
@@ -18,10 +18,10 @@ jobs:
   smoke-test:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v2.5.0
```

Comment:
While v2.5.0 is the latest release by this repo's calendar date, @v3+ is the one that switches to nodejs 16 https://github.com/actions/checkout/releases/tag/v3.0.0

@v2 still uses nodejs12 https://github.com/actions/checkout/blob/e2f20e631ae6d7dd3b768f56a5d2af784dd54791/action.yml#L74-L75

Reply:
thank Bryan, applied your comments